HHS/Head-Start-TTADP
# [TTAHUB-2907] Add "something went wrong" page

## Description of change

This adds a "something went wrong" context page to the HUB. 

Currently its hooked up to the retrieval on AR, Event, Session, and Goals.

## How to test

- Review the code.
- Ensure we match the UX design.
- Make sure we correctly display the relevant page for each error code type (404 vs 500) etc.
- Test by changing handlers to throw various response codes.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2907


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
